ssi-vc/src/lib.rs

```diff
@@ -2925,7 +2925,9 @@ _:c14n0 <https://w3id.org/security#verificationMethod> <https://example.org/foo/
         assert_eq!(n_proofs, 4);
     }
 
+    // Can't deserialise the URIs as verification methods (?)
```

Comment:
this relies on the `testing` flag for `ssi-dids` to be enabled (see `ssi-dids/src/lib.rs#973`)

Reply:
cheers